service/user/src/main/resources/application.yml

```
@@ -1,26 +1,3 @@
-  #        use_sql_comments: true # 콘솔에 표시되는 쿼리문 위에 어떤 실행을 하려는지 HINT 표시
-
-server:
-  port: 19092
-
-eureka:
-  client:
-    service-url:
-      defaultZone: ${EUREKA_SERVER_URL}
\ No newline at end of file
+  profiles:
+    active: local
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)